v4l2-request-test.c

```diff
@@ -257,7 +266,26 @@ int main(int argc, char *argv[])
 	setup_config(&config);
 
 	while (1) {
-		opt = getopt(argc, argv, "v:m:d:D:s:f:P:ilqh");
+	        int option_index = 0;
+		static struct option long_options[] = {
```

Comment:
Please fix the following block's alignment to one 3 tabs (1 more than where the declaration starts).

Reply:
if i got you right, I need to remove all whitespace and put one extra tab?

Reply:
Alignment looks good now but you missed a few whitespaces before `},`